<launch>
    <param name="package_path" value="$(find lcr_cal)"/>
    <rosparam file="$(find lcr_cal)/config/ros-node-config.yaml" command="load"/>
    <rosparam file="$(find lcr_cal)/config/setup-caliboard.yaml" command="load"/>

    <param name="/shared_parameter/center_x" value="0" />
    <param name="/shared_parameter/center_y" value="0" />
    <param name="/shared_parameter/center_z" value="0" />
    <param name="/shared_parameter/length_x" value="0" />
    <param name="/shared_parameter/length_y" value="0" />
    <param name="/shared_parameter/length_z" value="0" />
    <param name="/shared_parameter/rotate_x" value="0" />
    <param name="/shared_parameter/rotate_y" value="0" />
    <param name="/shared_parameter/rotate_z" value="0" />
    <param name="/shared_parameter/filter_distance" value="0" />
    <param name="/shared_parameter/filter_snr" value="0" />
    <param name="/shared_parameter/max_snr_points_num" value="0" />
    <param name="/shared_parameter/valid_distance_threshold" value="0" />
    <param name="shared_parameter" value="value" />
    <node pkg="lcr_cal" type="pointcloud_process_node" name="pointcloud_process_node" output="screen"/>
    <node pkg="lcr_cal" type="image_process_node" name="image_process_node" output="screen"/>
    <node pkg="lcr_cal" type="lc_calibration_node" name="lc_calibration_node" output="screen"/>
    <node pkg="lcr_cal" type="controller_menu" name="controller_menu" output="screen"  launch-prefix="gnome-terminal -- /bin/sh -c"/>

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find lcr_cal)/rviz/lc_calibration.rviz" output="screen"/>
    
</launch>